### ISSUE TITLE
Should we cut the `homepage` column from the main mod list

### ISSUE THREAD

**Issue by plague006** [KSP-CKAN/CKAN#1271]
* It's *very* far to the right. That might sound a little silly, but if you're trying to get an at-a-glance look at all your mods, it's usually not to compare their homepages.
* People will usually click on a mod they're interested in to read the description.
* The link is non-functional. A single click does nothing and a double click selects the mod to be installed.

Would love to hear input (especially from anyone who does use it).

As an aside: I used to use the homepages list to find mods that didn't have homepages set, but after #1254 is in a release, the only mod left without a homepage will be `z_DEPRECATED...`

**Comment by Murdabenne** [comment]
In all honesty I didnt even know it was there until I accidentally scrolled over about a week ago.  I use the right side pane for that info.  Presenting it on the main list seems a bit redundant, and I thing the versioning information would be better in its place.  Same goes for the description - The side panel is so much easier to use.

**Comment by plague006** [comment]
Closing in favor of #1270
